frontend/src/metabase-types/api/mocks/settings.ts

```diff
@@ -116,6 +116,7 @@ export const createMockSettings = (opts?: Partial<Settings>): Settings => ({
   "email-configured?": false,
   "enable-embedding": false,
   "enable-nested-queries": true,
+  "enable-query-caching": undefined,
```

Comment:
Should we just use `false` here?

Reply:
The setting is not present in an OSS build, so it would be `undefined` there. That's why I decided to use that as a default.